assets/js/googlesitekit/modules/create-module-store.js

```diff
@@ -68,8 +68,9 @@ export const createModuleStore = ( slug, {
 	validateCanSubmitChanges = undefined,
 } = {} ) => {
 	invariant( slug, 'slug is required.' );
+	invariant( storeName, 'storeName is required.' );
 
-	storeName = storeName || `modules/${ slug }`;
+	storeName = storeName || `test/${ slug }`; // TODO: remove this now that the storeName is required
```

Comment:
Again, no more default so the `||` logic and fallback should be removed, as per the comment you added.